main.go

```diff
@@ -1,12 +1,148 @@
 package main
 
+import (
+	"fmt"
+	"os"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/microkit/command"
+	microserver "github.com/giantswarm/microkit/server"
+	"github.com/giantswarm/microkit/transaction"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/microstorage"
+	"github.com/giantswarm/microstorage/memory"
+	"github.com/spf13/viper"
+
+	"github.com/giantswarm/cluster-operator/flag"
+	"github.com/giantswarm/cluster-operator/server"
+	"github.com/giantswarm/cluster-operator/service"
+)
+
+const (
+	notAvailable = "n/a"
+)
+
 var (
-	description string = "The cluster-operator is an in-cluster agent that handles Giant Swarm guest cluster specific resources."
-	gitCommit   string = "n/a"
-	name        string = "cluster-operator"
-	source      string = "https://github.com/giantswarm/cluster-operator"
+	description            = "The cluster-operator is an in-cluster agent that handles Giant Swarm guest cluster specific resources."
+	f           *flag.Flag = flag.New()
```

Comment:
Nitpick: do we need this type `*flag.Flag` here? I would remove or retain all types.